fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -402,130 +404,37 @@ public void testCreateVersionWithSlugHeader() throws Exception {
-            assertEquals("Didn't get a BAD_REQUEST response!", BAD_REQUEST.getStatusCode(), getStatus(response));
-
-            // Request must fail with constrained exception due to empty body
-            assertConstrainedByPresent(response);
-        }
-    }
-
-    @Ignore //TODO Fix this test
-    @Test
-    public void testCreateContainerWithoutServerManagedTriples() throws Exception {
-        createVersionedContainer(id);
-
-        final HttpPost createMethod = postObjMethod(id + "/" + FCR_VERSIONS);
-        createMethod.addHeader(CONTENT_TYPE, N3);
-        createMethod.setEntity(new StringEntity("<" + subjectUri + "> <info:test#label> \"part\""));
-        createMethod.addHeader(MEMENTO_DATETIME_HEADER, MEMENTO_DATETIME);
-
-        // Attempt to create memento with partial record
-        try (final CloseableHttpResponse response = execute(createMethod)) {
-            assertEquals("Didn't get a BAD_REQUEST response!", BAD_REQUEST.getStatusCode(), getStatus(response));
-
-            // Request must fail with constrained exception due to empty body
-            assertConstrainedByPresent(response);
-        }
-    }
-
-    /**
-     * POST to create LDPCv without memento-datetime must ignore body
-     *
-     * @throws Exception in case of error with test
-     */
-    @Ignore //TODO Fix this test
-    @Test
-    public void testCreateVersionWithBody() throws Exception {
-        createVersionedContainer(id);
-
-        final String mementoUri = createContainerMementoWithBody(subjectUri, null);
-        assertMementoUri(mementoUri, subjectUri);
-
-        final HttpGet httpGet = new HttpGet(mementoUri);
-        try (final CloseableDataset dataset = getDataset(httpGet)) {
-            final DatasetGraph results = dataset.asDatasetGraph();
-
-            final Node mementoSubject = createURI(subjectUri);
-
-            assertTrue("Memento created without datetime must retain original state",
-                    results.contains(ANY, mementoSubject, TEST_PROPERTY_NODE, createLiteral("foo")));
-            assertFalse("Memento created without datetime must ignore updates",
-                    results.contains(ANY, mementoSubject, TEST_PROPERTY_NODE, createLiteral("bar")));
-
-            // memento should be the same as original, in this case
-            assertMementoEqualsOriginal(mementoUri);
-        }
+        assertEquals("Created memento with Memento-Datetime!",
+                BAD_REQUEST.getStatusCode(), getStatus(post));
     }
 
-    @Ignore //TODO Fix this test
+    @Ignore("get memento not implemented")
     @Test
     public void testCreateVersionWithDatetimeAndBody() throws Exception {
         createVersionedContainer(id);
 
-        final String mementoUri = createContainerMementoWithBody(subjectUri, MEMENTO_DATETIME);
+        final String mementoUri = createContainerMementoWithBody(subjectUri);
+//        final String mementoUri = createContainerMementoWithBody(subjectUri, MEMENTO_DATETIME);
```

Comment:
I accidently hit the button that resolved the previous your previous comment when I was trying to reply...

I commented out these lines because often the context of the original test makes it easier to understand what they're supposed to be testing, making them easier to recreate. Personally, I'd prefer to keep the commented out lines incase they proves useful when I'm working on reenabling them, but will remove them if you insist.

Reply:
I removed the commented out lines